### PR TITLE
[CHF-572] Added Health Check Implementation for Zen Thermostat.

### PR DESCRIPTION
**Pull request by skt123**
@jackchi @ShunmugaSundar 
Added Health check for Zen Thermostat with checkInterval of 12 mins.
Please review and merge.

### CODE REVIEW

File: devicetypes/zenwithin/zen-thermostat.src/zen-thermostat.groovy [hunk 1]
```diff
@@ -511,7 +517,7 @@ def configure()
       	  //Thermostat - Cluster 201
           "zdo bind 0x${device.deviceNetworkId} 1 1 0x201 {${device.zigbeeId}} {}", "delay 500",
           
-          "zcl global send-me-a-report 0x201 0 0x29 5 300 {3200}", 
+          "zcl global send-me-a-report 0x201 0 0x29 0 300 {3200}",
```
Comment (jackchi) [inline devicetypes/zenwithin/zen-thermostat.src/zen-thermostat.groovy]: We do not want to change the `minReportTime` attribute in this zcl cmd. 
FYI:
`minReportTime - Minimum number of seconds between reports`

**Review by jackchi — CHANGES_REQUESTED** [review]

File: devicetypes/zenwithin/zen-thermostat.src/zen-thermostat.groovy (hunk 1 above)
Comment (jackchi) [reply devicetypes/zenwithin/zen-thermostat.src/zen-thermostat.groovy]: https://github.com/PhysicalGraph/appengine-zigbee/blob/master/src/main/groovy/physicalgraph/zigbee/Zigbee.groovy#L348

**Comment by skt123** [comment]
@jackchi 
Made the required changes. Please check.

**Review by jackchi — APPROVED** [review]
👍

**Pull request merged**